### ISSUE TITLE
ATM6 v1.1.4 and v.1.1.5 fail to start on newer versions of openjdk

### ISSUE THREAD

**Issue by sanyi** [AllTheMods/ATM-6#181]
**Describe the bug**
Newer versions of  ATM6 v.1.1.4 and v.1.1.5 fail during loading of the game  (on Ubuntu)  with openjdk 13.0.3 and 14.0.1,  while it loads without problems with 1.8.0_265 and 11.0.8. Previous versions worked well with openjdk 14.0.1 too.

The problems seems to be related to biggerreactors and phosphophyllite.
If needed I can provide you with extended logs. 

**Screenshots**
![image](https://user-images.githubusercontent.com/2527907/96365712-38e9d500-114b-11eb-83e1-2fdbc920c5f0.png)

```-- Head --
Thread: Render thread
Stacktrace:
	at net.roguelogix.phosphophyllite.config.ConfigLoader.<clinit>(ConfigLoader.java:165) ~[?:0.0.2-beta] {re:classloading}
-- MOD biggerreactors --
Details:
	Caused by 0: java.lang.ExceptionInInitializerError
		at net.roguelogix.phosphophyllite.config.ConfigManager.loadConfig(ConfigManager.java:116) ~[?:0.0.2-beta] {re:classloading}
		at net.roguelogix.phosphophyllite.config.ConfigManager.registerConfig(ConfigManager.java:27) ~[?:0.0.2-beta] {re:classloading}
		at net.roguelogix.phosphophyllite.registry.Registry.registerConfigs(Registry.java:586) ~[?:0.0.2-beta] {re:classloading}
		at net.roguelogix.phosphophyllite.registry.Registry.onModLoad(Registry.java:170) ~[?:0.0.2-beta] {re:classloading}
		at net.roguelogix.biggerreactors.BiggerReactors.<init>(BiggerReactors.java:30) ~[?:0.1.5-beta] {re:classloading}
		at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {re:classloading,re:classloading,re:classloading}
		at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:?] {re:classloading,re:classloading,re:classloading}
		at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?] {re:classloading,re:classloading,re:classloading}
		at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500) ~[?:?] {}
		at java.lang.reflect.ReflectAccess.newInstance(ReflectAccess.java:124) ~[?:?] {}
		at jdk.internal.reflect.ReflectionFactory.newInstance(ReflectionFactory.java:346) ~[?:?] {re:classloading,re:classloading,re:classloading}
		at java.lang.Class.newInstance(Class.java:604) ~[?:?] {}
		at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:81) ~[?:34.1] {re:classloading}
		at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:111) ~[?:?] {re:classloading}
		at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1800) ~[?:?] {}
		at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1792) ~[?:?] {}
		at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598) ~[?:?] {}
		at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177) ~[?:?] {}

	Mod File: biggerreactors-1.16.3-0.1.5-beta.jar
	Failure message: Bigger Reactors (biggerreactors) has failed to load correctly
		java.lang.ExceptionInInitializerError: null
	Mod Version: 0.1.5
	Mod Issue URL: https://github.com/RogueLogix/BiggerSeries/issues/
	Exception message: java.lang.NullPointerException
```

**Additional context**
I am currently launching the game with MultiMC 5.  I am filling this bug mainly because someone might want this future proofed also as a possible solution for other linux users.

**Comment by nameless312** [comment]
Have the exact same issue, im running manjaro, i tried jdk 8 but it didnt seem to work very well either, jdk 15 wont even boot

**Comment by RogueLogix** [comment]
Use Java 8, I (BiggerReactors and Phosphophyllite dev) only test with Java 8, and this is a known issue with newer versions of java and my mod.

RogueLogix/BiggerSeries#11

The problem is that i reflect into the Java Runtime, and it changed in newer versions such that i can't reflect in the same way.

**Comment by BedrockLegends** [comment]
Yeah Rogue already told us that they pretty much wont fix it, so anyone who use bigger reactors will have to use Java 8

**Comment by RogueLogix** [comment]
at some point i will, but its low priority and hard to verify a fix for until Mojang/Forge update to something newer than Java 8

**Comment by RogueLogix** [comment]
I guess i can correct myself with this, its a solution to a problem that doesn't actually exist causing this issue.
I dug into why it was happening, and it turns out there are two overrides for it, one of which does work in newer java versions too.
Phos 0.0.3 will have that fixed, expect that later today

**Comment by whatthedrunk** [comment]
This should be fixed in 1.1.6 which will be out later today. Let us know how it goes.

**Comment by sanyi** [comment]
I confirm that ATM6-1.1.6 starts (also loads a previous save game) with openjdk 14.0.1. Thank you all!
